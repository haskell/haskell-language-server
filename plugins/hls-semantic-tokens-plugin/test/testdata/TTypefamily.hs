{-# LANGUAGE TypeFamilies #-}
module TTypefamily where

type family Foo a where
  Foo Int = Int
  Foo a = String
