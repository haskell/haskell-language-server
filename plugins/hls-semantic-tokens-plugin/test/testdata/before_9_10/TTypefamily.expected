4:13-16 TTypeFamily "Foo"
4:17-18 TTypeVariable "a"
5:3-6 TTypeFamily "Foo"
5:7-10 TTypeConstructor "Int"
5:13-16 TTypeConstructor "Int"
6:3-6 TTypeFamily "Foo"
6:7-8 TTypeVariable "a"
6:11-17 TTypeSynonym "String"
